src/Kdyby/Console/DI/ConsoleExtension.php

```diff
@@ -81,7 +81,8 @@ public function loadConfiguration()
 
 		Nette\Utils\Validators::assert($config, 'array');
 		foreach ($config['commands'] as $command) {
-			$def = $builder->addDefinition($this->prefix('command.' . md5(Nette\Utils\Json::encode($command))));
+			$entityName = ($command instanceof Statement) ? $command->getEntity() : '';
+			$def = $builder->addDefinition($this->prefix('command.' . md5($entityName.Nette\Utils\Json::encode($command))));
```

Comment:
Agreed! I think it might have been premature prevention of possible conflicts with master service container (which was my first thought and worry.) After testing though, all seems ok and working no matter what I throw at it (combined `services` alongside with `console` section, and DI seems happy.)

Updated, squashed, `push --force`d the commit and now we should be good to go :)